lib/alerts_viewer_web/live/stop_recommendation_algorithm/alert_duration_component.ex

```diff
@@ -0,0 +1,51 @@
+defmodule AlertsViewer.StopRecommendationAlgorithm.AlertDurationComponent do
+  @moduledoc """
+  Component for controlling the Max adherence delay alert recommendation algorithm.
```

Comment:
nitpick: I think this description could be updated.

Reply:
whups >_>